### PR TITLE
add guide for scheduler gpu optimization

### PR DESCRIPTION
**Pull request by VenkateshJaya**
Contributes to #9 

Adds a guide to https://docs.rapids.ai/deployment/stable/guides/

- creates a cluster on GKE with an additional T4 gpu nodepool labelled as dedicated for the scheduler
- installs drivers and Dask Operator
- creates a Dask cluster where the scheduler is placed on a node with the T4 gpu using node affinity

### CODE REVIEW

File: source/guides/scheduler-gpu-optimization.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+With this command, you’ve launched a GKE cluster called `rapids-gpu`. You’ve specified that it should use nodes of type
+a2-highgpu-2g, each with two A100 GPUs.
+
+## Create the dedicated nodepool for the scheduler
+
+Now create a new nodepool on this GPU cluster.
+
+```console
+$ gcloud container node-pools create scheduler-pool --cluster rapids-gpu \
+  --accelerator type=nvidia-tesla-p4,count=1 --machine-type n1-standard-2 \
```
Comment (jacobtomlinson) [inline source/guides/scheduler-gpu-optimization.md:40]:
```suggestion
  --accelerator type=nvidia-tesla-t4,count=1 --machine-type n1-standard-2 \
```

File: source/guides/scheduler-gpu-optimization.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+          port: 8787
+          targetPort: "http-dashboard"
+```
+
+You can create this cluster with `kubectl`.
+
+```console
+$ kubectl apply -f rapids-dask-cluster.yaml
+```
+
```
Comment (jacobtomlinson) [inline source/guides/scheduler-gpu-optimization.md:209]: Could you also add an example that uses the Python API?

You'll want to use `make_cluster_spec` and then add the affinity section to the dictionary.

https://kubernetes.dask.org/en/latest/operator_kubecluster.html#customising-your-cluster

**Review by jacobtomlinson — COMMENTED** [review]
This is a great first draft! I left a few comments that can help take this to the next level.

File: source/guides/scheduler-gpu-optimization.md (hunk 2 above)
Comment (VenkateshJaya) [reply source/guides/scheduler-gpu-optimization.md:209]: I added the section which specifies on how to use KubeCluster for the same.

File: source/guides/scheduler-gpu-optimization.md [hunk 3]
```diff
@@ -0,0 +1,357 @@
+# GPU optimization for the Dask scheduler
```
Comment (jacobtomlinson) [inline source/guides/scheduler-gpu-optimization.md:1]: Could you add Kubernetes to this title?

File: source/guides/index.md [hunk 4]
```diff
@@ -39,6 +39,17 @@ Guidance on Dask scheduler software and hardware requirements.
 {bdg-primary}`Dask`
 ````
 
+````{grid-item-card}
+:link: scheduler-gpu-optimization
+:link-type: doc
+Optimizing Dask Scheduler
```
Comment (jacobtomlinson) [inline source/guides/index.md:45]: Could you add Kubernetes to this title?

File: source/guides/index.md [hunk 5]
```diff
@@ -39,6 +39,17 @@ Guidance on Dask scheduler software and hardware requirements.
...
+^^^
+Use a T4 for the scheduler to optimize resource costs
```
Comment (jacobtomlinson) [inline source/guides/index.md:47]: Could you mention Kubernetes in here?

File: source/guides/index.md [hunk 6]
```diff
@@ -39,6 +39,17 @@ Guidance on Dask scheduler software and hardware requirements.
...
+
+{bdg-primary}`Dask`
+{bdg-primary}`Kubernetes`
```
Comment (jacobtomlinson) [inline source/guides/index.md:50]:
```suggestion
{bdg-primary}`Kubernetes`
{bdg-primary}`dask-operator`
```

**Review by jacobtomlinson — COMMENTED** [review]
Just a couple of suggestions about titles and tags, then this is good to merge!

**Review by jacobtomlinson — APPROVED** [review]

**Pull request merged**